### PR TITLE
[semver:major] Initial release

### PR DESCRIPTION
**Pull request by gmemstr**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -23,71 +19,65 @@ parameters:
...
 
 jobs:
-  # Define one or more jobs which will utilize your orb's commands and parameters to validate your changes.
-  integration-test-1:
-    docker:
-      - image: cimg/base:stable
+  integration-test_using-sln:
+    executor:
+      name: win/default
+      shell: powershell.exe
```
Comment (KyleTryon) [inline .circleci/config.yml]: Wouldnt the shell be set at the command level for the orb so the user isnt tasked with this?

File: .circleci/config.yml [hunk 2]
```diff
@@ -23,71 +19,65 @@ parameters:
...
-          dir: ./src/scripts
-          exclude: SC2148
-      # optional: Run BATS tests against your scripts
-      - bats/run:
-          path: ./src/tests
-      # Publish development version(s) of the orb.
       - orb-tools/publish-dev:
           orb-name: circleci/microsoft-msix
-          context: orb-publishing # A restricted context containing your private publishing credentials. Will only execute if approved by an authorized user.
+          context: orb-publishing
```
Comment (KyleTryon) [inline .circleci/config.yml]: The correct context name is: `orb-publisher`

File: src/commands/sign.yml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+parameters:
+  sdk-version:
+    type: string
+    default: "10.0.17763.0"
+    description: Windows SDK version to use when invoking Makeappx.exe.
+  working-dir:
+    type: string
+    default: "~/project"
```
Comment (KyleTryon) [inline src/commands/sign.yml:8]: Is this correct for windows?

File: src/@orb.yml [hunk 4]
```diff
@@ -2,14 +2,10 @@ version: 2.1
 
 description: >
   Sample orb description
```
Comment (KyleTryon) [inline src/@orb.yml]: Needs description

File: src/@orb.yml [hunk 5]
```diff
@@ -2,14 +2,10 @@ version: 2.1
...
-# What will your orb allow users to accomplish?
-# Descriptions should be short, simple, and informative.
 
-# This information will be displayed in the orb registry and is not mandatory.
 display:
   home_url: "https://www.website.com/docs"
```
Comment (KyleTryon) [inline src/@orb.yml]: needs home url

File: sample/SampleProject/My Project/Resources.resx [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
```
Comment (KyleTryon) [inline sample/SampleProject/My Project/Resources.resx:115]: I assume it is fine, but is this key of any importance?

File: .circleci/config.yml [hunk 7]
```diff
@@ -23,71 +19,65 @@ parameters:
...
-      # be skipped.
-      # e.g. [semver:patch] will cause a patch version to be published.
+      - integration-test_using-sln
+      - integration-test_using-exe
       - orb-tools/dev-promote-prod-from-commit-subject:
           orb-name: circleci/microsoft-msix
           context: orb-publishing
           add-pr-comment: false
           fail-if-semver-not-indicated: true
           publish-version-tag: false
```
Comment (KyleTryon) [inline .circleci/config.yml]: Enable tagging
  Add the ssh key with write access and the orb-publisher bot user.
enable PR commenting.
will slack confluence document.

File: .circleci/config.yml (hunk 2 above)
Comment (KyleTryon) [reply .circleci/config.yml]: ^ propagate to other locations in the config.

**Review by KyleTryon — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (gmemstr) [reply .circleci/config.yml]: Completely forgot this could be set at the step level.

File: sample/SampleProject/My Project/Resources.resx (hunk 6 above)
Comment (gmemstr) [reply sample/SampleProject/My Project/Resources.resx:115]: Not that I'm aware of.

File: src/commands/sign.yml (hunk 3 above)
Comment (gmemstr) [reply src/commands/sign.yml:8]: PowerShell does support it, although `cmd` does not :)

**Review by KyleTryon — APPROVED** [review]
Looking good 👍

**Pull request merged**